### DISCUSSION TITLE
1.28.5 teslamate container rebooting

### DISCUSSION THREAD

**Discussion by chrislombardo** [teslamate-org/teslamate#3816] [General]
Since upgrading from 1.28.3 to 1.28.5, the teslamate docker container constantly reboots.  The error message from docker log is:

```
Failed to create thread: Operation not permitted (1)
Aborted (core dumped)
```

Nothing has changed other than I did a docker pull for the new versions and followed the instructions to restore the database.  All other containers stay up.  I've used the same procedure (via a script) to upgrade for a very long time without any issues.

I pulled image teslamate:1.28.4 and have same issue. I then pulled teslamate:1.28.3 and the issue went away.

**Comment by micves** [comment]
You might need to upgrade docker. Check this one out: https://github.com/teslamate-org/teslamate/issues/3754

**Comment by chrislombardo** [comment]
Thank you for pointing that out.  I'm running docker 20.10.7 on Ubuntu 16.04.7 LTS.  Not sure there is a more recent version of docker available for this old OS.  Anyway, I added the following to docker-compose.yml and teslamate 1.28.5 started with no problem.

```
security_opt:
      - seccomp:unconfined
```
